config/operator/manager/manager.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: namespace
+    app.kubernetes.io/instance: clusterlink-operator-ns
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: cl-operator
+    app.kubernetes.io/part-of: cl-operator
+  name: clusterlink-operator-ns
```

Comment:
is the choice of adding `-ns` Kubebuilder or input from the user?

Reply:
I added it, I can remove it...